src/test/java/TypewiseAlert/TypewiseAlertTest.java

```diff
@@ -6,10 +6,32 @@
 
 public class TypewiseAlertTest 
 {
-    @Test
-    public void infersBreachAsPerLimits()
-    {
-      assertTrue(TypewiseAlert.inferBreach(12, 20, 30) ==
+   @Test
+  public void infersBreachAsPerLimits()
+  {
+    assertTrue("test", TypewiseAlert.inferBreach(12, 20, 30) ==
         TypewiseAlert.BreachType.TOO_LOW);
-    }
+    assertTrue("test", TypewiseAlert.inferBreach(22, 20, 30) ==
+        TypewiseAlert.BreachType.NORMAL);
+    assertTrue("test", TypewiseAlert.inferBreach(32, 20, 30) ==
+        TypewiseAlert.BreachType.TOO_HIGH);
+    
+    
+    assertTrue("test", TypewiseAlert.classifyTemperatureBreach(TypewiseAlert.CoolingType.PASSIVE_COOLING)[0] ==0);
+    assertTrue("test", TypewiseAlert.classifyTemperatureBreach(TypewiseAlert.CoolingType.HI_ACTIVE_COOLING)[1] ==45);
+    assertTrue("test", TypewiseAlert.classifyTemperatureBreach(TypewiseAlert.CoolingType.MED_ACTIVE_COOLING)[1] ==40);
+ 
+    assertTrue("test", TypewiseAlert.sendAlert(TypewiseAlert.AlertTarget.TO_CONTROLLER,TypewiseAlert.BreachType.TOO_HIGH));
```

Comment:
How do we verify that the controller has received the intended TOO_HIGH alert message? test cases need to verify that too.